ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2113,3 +2120,102 @@ - (BOOL)isEqual:(id)object {
 }
 
 @end
+
+
+#pragma mark - ORKLocationAnswerFormat
+
+@implementation ORKLocationAnswerFormat
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        _useCurrentLocation = YES;
+    }
+    return self;
+}
+
+- (BOOL)isAnswerValidWithString:(nullable NSString *)text {
+    BOOL isValid = [super isAnswerValidWithString:text];
+    
+    MKPlacemark *placemark = [MKPlacemark ork_placemarkWithString:text];
+    
+    if (placemark && CLLocationCoordinate2DIsValid(placemark.coordinate)) {
```

Comment:
Why the coordinate obtained from a placemark need to be validated? Shouldn't `MKPlacemark` always base on a valid coordinate?